### PR TITLE
Added changie logs for v2.2.0

### PR DESCRIPTION
**Pull request by cchen-vertica**
We haven't added the release logs for the features in v2.2.0. This PR added the major logs.

### CODE REVIEW

**Review by fenic-fawkes — APPROVED** [review]

File: changes/unreleased/Added-20240808-140515.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+kind: Added
```
Comment (roypaulin) [inline changes/unreleased/Added-20240808-140515.yaml:1]: This is not a highlight

File: changes/unreleased/Added-20240808-140747.yaml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+kind: Added
```
Comment (roypaulin) [inline changes/unreleased/Added-20240808-140747.yaml:1]: Too much details. Only the main feature should be highlighted

File: changes/unreleased/Added-20240808-141400.yaml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+kind: Added
```
Comment (roypaulin) [inline changes/unreleased/Added-20240808-141400.yaml:1]: Not a highlight

File: changes/unreleased/Added-20240808-141816.yaml [hunk 4]
```diff
@@ -0,0 +1,6 @@
+kind: Added
```
Comment (roypaulin) [inline changes/unreleased/Added-20240808-141816.yaml:1]: Technical details are not needed on the release note

File: changes/unreleased/Changed-20240808-140420.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+kind: Changed
```
Comment (roypaulin) [inline changes/unreleased/Changed-20240808-140420.yaml:1]: Only features exposed to users must appear in the release note

File: changes/unreleased/Changed-20240808-141635.yaml [hunk 6]
```diff
@@ -0,0 +1,6 @@
+kind: Changed
```
Comment (roypaulin) [inline changes/unreleased/Changed-20240808-141635.yaml:1]: Not a highlight

File: changes/unreleased/Fixed-20240808-142106.yaml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+kind: Fixed
```
Comment (roypaulin) [inline changes/unreleased/Fixed-20240808-142106.yaml:1]: I thought I had already added this.

File: changes/unreleased/Added-20240808-140515.yaml (hunk 1 above)
Comment (cchen-vertica) [reply changes/unreleased/Added-20240808-140515.yaml:1]: Should I remove this changie if it's not important/hightlight?

File: changes/unreleased/Added-20240808-141816.yaml (hunk 4 above)
Comment (cchen-vertica) [reply changes/unreleased/Added-20240808-141816.yaml:1]: Those are the fields the customer will see in the status, are those the technical details?

File: changes/unreleased/Changed-20240808-140420.yaml (hunk 5 above)
Comment (cchen-vertica) [reply changes/unreleased/Changed-20240808-140420.yaml:1]: Sure, will remove it.

File: changes/unreleased/Changed-20240808-141635.yaml (hunk 6 above)
Comment (cchen-vertica) [reply changes/unreleased/Changed-20240808-141635.yaml:1]: Will remove it.

File: changes/unreleased/Fixed-20240808-142106.yaml (hunk 7 above)
Comment (cchen-vertica) [reply changes/unreleased/Fixed-20240808-142106.yaml:1]: I didn't see the changie in the PR 835 and the release of 2.1.3.

**Pull request merged**